frontend/components/MainContent/_styles.scss

```diff
@@ -6,5 +6,8 @@
   // of the main-content when there is a banner. (e.g. sandbox mode)
   // Without it the main content pushes the banner off the page.
   overflow: auto;
-  animation: fade-in 250ms ease-out;
+
+  > :not(.main-content--animation-disabled) {
```

Comment:
Open for other solutions to not make the banner flash on page change

tldr: 
`main-content` has `animation: fade-in 250ms ease-out;` applied to it, but looks super weird to have that banner flash over and over on each page click since it's not moving (feels not React like at all!), so I removed the animation from the banners only by putting it in a div wrapper called `.main-content--animation-disabled`

Reply:
see above